src/boot_O2_g3/fault_drawer.c

```diff
@@ -249,17 +294,17 @@ void FaultDrawer_DrawText(s32 x, s32 y, const char* fmt, ...) {
 }
 
 void FaultDrawer_SetDrawerFB(void* fb, u16 w, u16 h) {
```

Comment:
Since `Fault_SetFB` -> `Fault_SetFrameBuffer` and `Fault_CommitFB` -> `Fault_DisplayFrameBuffer`, would `FaultDrawer_SetDrawerFB` -> `FaultDrawer_SetDrawerFrameBuffer` make sense?

Similarly, would `void* fb` -> `void* frameBuffer` make sense?

Reply:
I kept this name because that's how it is named in OoT, but you are right I'll rename it